src/test/java/com/bags/simulacrum/Weapon/WeaponModifierTest.java

```diff
@@ -30,7 +34,6 @@
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
-        subject = new WeaponModifier();
```

Comment:
With Mockito in play you don't want to setup your subject under test, Mockito will automatically do that for you.